cirq-google/cirq_google/engine/qcs_notebook_test.py

```diff
@@ -17,11 +17,13 @@
 
 
 def test_get_device_sampler():
-    result = get_qcs_objects_for_notebook()
-    assert result.device is cg.Sycamore
+    result = get_qcs_objects_for_notebook('not_a_valid_project_name')
```

Comment:
I think it'd be good for the tests to continue to exercise the no-arguments call variant. Perhaps we can do both? (We already exercise passing both arguments on line 28 below.)